<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>878</width>
    <height>641</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="drawspace" name="space" native="true"/>
      </item>
      <item>
       <widget class="QFrame" name="frame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <widget class="QPushButton" name="colour_button">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>110</y>
           <width>161</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Choose colour</string>
         </property>
        </widget>
        <widget class="QLabel" name="move_info">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>201</width>
           <height>81</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Hold mouse left button to move fractal.&lt;/p&gt;&lt;p&gt;Zoom fractal (with shift to center) &lt;/p&gt;&lt;p&gt;	with mouse wheel.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
        <widget class="QLabel" name="iter_info">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>180</y>
           <width>161</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Iterations</string>
         </property>
        </widget>
        <widget class="QLabel" name="colour_info">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>240</y>
           <width>161</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Colour steps</string>
         </property>
        </widget>
        <widget class="QPushButton" name="settings_button">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>310</y>
           <width>161</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Apply settings</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="iter_box">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>200</y>
           <width>161</width>
           <height>22</height>
          </rect>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10000</number>
         </property>
        </widget>
        <widget class="QSpinBox" name="colour_box">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>260</y>
           <width>161</width>
           <height>22</height>
          </rect>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10000</number>
         </property>
        </widget>
        <widget class="QPushButton" name="reset_button">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>511</y>
           <width>161</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>RESET</string>
         </property>
        </widget>
        <widget class="QPushButton" name="reset_settings">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>350</y>
           <width>161</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Resert setting</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>878</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>drawspace</class>
   <extends>QWidget</extends>
   <header>drawspace.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>colour_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>choose_colour()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>741</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>874</x>
     <y>41</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>settings_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>set_settings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>798</x>
     <y>362</y>
    </hint>
    <hint type="destinationlabel">
     <x>872</x>
     <y>332</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>reset_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>reset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>803</x>
     <y>556</y>
    </hint>
    <hint type="destinationlabel">
     <x>872</x>
     <y>531</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>reset_settings</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>reset_settings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>797</x>
     <y>394</y>
    </hint>
    <hint type="destinationlabel">
     <x>872</x>
     <y>396</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>choose_colour()</slot>
  <slot>set_settings()</slot>
  <slot>reset()</slot>
  <slot>reset_settings()</slot>
 </slots>
</ui>
